app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,9 @@
 <resources>
     <string name="app_name">Pokemon</string>
+    <string name="regions">Regions</string>
+    <string name="pokemons">Pokemons</string>
+    <string name="favorite">Favorited</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.hello_blank_fragment` appears to be unused
 Unused resources make applications larger and slow down builds.